docs/docs/05-reusable-components.md

```diff
@@ -239,6 +239,23 @@ Counter.propTypes = { initialCount: React.PropTypes.number };
 Counter.defaultProps = { initialCount: 0 };
 ```
 
+Alternatively, you can define `propTypes` and `defaultProps` as static properties of the class.
```

Comment:
I'm suggesting to reword as `Alternatively, you can define `propTypes` and `defaultProps` using static getters.` but it's better if someone from React core team will chime in